README.md

```diff
@@ -2,179 +2,86 @@
-
-NAME                                            DESIRED   CURRENT   AGE
-statefulset.apps/bar-pravega-segmentstore       3         3         2m
-```
-
-By default, a `PravegaCluster` instance is only accessible within the cluster through the Controller `ClusterIP` service. From within the Kubernetes cluster, a client can connect to Pravega at:
-
-```
-tcp://[CLUSTER_NAME]-pravega-controller.[NAMESPACE]:9090
-```
-
-And the `REST` management interface is available at:
+### Upgrade the Operator
 
-```
-http://[CLUSTER_NAME]-pravega-controller.[NAMESPACE]:10080/
-```
+For upgrading the pravega operator check the document [operator-upgrade](doc/operator-upgrade.md).
 
-Check out the [external access documentation](doc/external-access.md) if your clients need to connect to Pravega from outside Kubernetes.
+### Installing on minikube
 
-Check out the [exposing Segmentstore service on single IP address](https://github.com/pravega/pravega-operator/blob/4aa88641c3d5a1d5afbb2b9e628846639fd13290/doc/external-access.md#exposing-segmentstore-service-on-single-ip-address-and-different-ports) if your clients need to connect to Pravega Segment store on the same IP address from outside Kubernetes.
+For installing operator and cluster on minikbe, refer to [minikube](doc/minikube_setup.md).
 
-### Scale a Pravega cluster
+### Script for complete Pravega Installation
 
-You can scale Pravega components independently by modifying their corresponding field in the Pravega resource spec. You can either `kubectl edit` the cluster or `kubectl patch` it. If you edit it, update the number of replicas for Controller, and/or Segment Store and save the updated spec.
+For installing pravega and all the components together, please refer to [this](setup/README.md).
 
-Example of patching the Pravega resource to scale the Segment Store instances to 4.
+### Scale the Cluster
 
-```
-kubectl patch PravegaCluster [CLUSTER_NAME] --type='json' -p='[{"op": "replace", "path": "/spec/pravega/segmentStoreReplicas", "value": 4}]'
-```
+For scaling the parvega cluster using helm, refer to [this](charts/pravega#upgrading-pravega-cluster).
 
-### Upgrade a Pravega cluster
+### Upgrade the cluster
 
 Check out the [upgrade guide](doc/upgrade-cluster.md).
 
-### Uninstall the Pravega cluster
-
-```
-$ helm uninstall [PRAVEGA_RELEASE_NAME]
-$ kubectl delete -f ./example/pvc-tier2.yaml
-```
-
-### Uninstall the Operator
-
-> Note that the Pravega clusters managed by the Pravega operator will NOT be deleted even if the operator is uninstalled.
-
-```
-$ helm uninstall [PRAVEGA_OPERATOR_RELEASE_NAME]
-```
-
-If you want to delete the Pravega clusters, make sure to do it before uninstalling the operator.
-
-### Manual installation
-
-You can also manually install/uninstall the operator and Pravega with `kubectl` commands. Check out the [manual installation](doc/manual-installation.md) document for instructions.
-
 ## Configuration
```

Comment:
Please remove this section. This is **not** relevant for operator as it is _Pravega Configuration._ and this page is about p-operator.
We can link to the Configuration document as part of Deploying Pravega document.